llama-index-integrations/indices/llama-index-indices-managed-dashscope/llama_index/indices/managed/dashscope/transformations.py

```diff
@@ -0,0 +1,191 @@
+"""
+This module maintains the list of transformations that are supported by the system.
+"""
+
+from enum import Enum
+from typing import Generic, Sequence, Type, TypeVar
+
+from llama_index.core.bridge.pydantic import (
+    BaseModel,
+    Field,
+    GenericModel,
+    ValidationError,
+)
+
+from llama_index.core.schema import BaseComponent, BaseNode, Document
+
+
+# Transform Input/Output Types
+class TransformationIOType(BaseModel):
```

Comment:
suspiciously similar to other code in the codebase 😉

Reply:
I fix it by importing necessary class from llama_index.core.ingestion.transformations, but build_configurable_transformation_enum and ConfiguredTransformation still need to be rewriten.

Reply:
oh nothing to fix haha was just making a joke